opensrp-reveal/src/main/java/org/smartregister/reveal/fragment/StatsFragment.java

```diff
@@ -0,0 +1,97 @@
+package org.smartregister.reveal.fragment;
+
+import android.app.ProgressDialog;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.v4.app.Fragment;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import org.smartregister.reveal.R;
+import org.smartregister.reveal.contract.StatsFragmentContract;
+import org.smartregister.reveal.presenter.StatsFragmentPresenter;
+
+import java.util.Map;
+
+import static org.smartregister.reveal.util.Constants.SyncInfo.SYNCED_CLIENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.SYNCED_EVENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.TASK_UNPROCESSED_EVENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.UNSYNCED_CLIENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.UNSYNCED_EVENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.VALID_CLIENTS;
+import static org.smartregister.reveal.util.Constants.SyncInfo.VALID_EVENTS;
+
+public class StatsFragment extends Fragment implements StatsFragmentContract.View {
+    private StatsFragmentPresenter presenter;
+
+    private ProgressDialog progressDialog;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Perhaps 'progressDialog' could be replaced by a local variable.](https://app.codacy.com/app/OpenSRP/opensrp-client-reveal/pullRequest?prid=4587928)